### ISSUE TITLE
RippleRestoreTask part of Ripple

### ISSUE THREAD

**Issue by danielmarbach** [FubuMvcArchive/ripple#248]
Would it be possible to make

https://github.com/Particular/RippleRestoreTask

part of ripple? Or at least point the documentation to the task? I think it is really handy.

**Comment by danielmarbach** [comment]
Any updates on this?

**Comment by jmarnold** [comment]
We need to consolidate the changes from ripple v3. I'm targeting next Friday (January 10th) as the tentative release for ripple 3. Once that's settled, we can more easily work this in.

The task itself wasn't the issue, if I call correctly. It was an issue of how the NServiceBus guys needed the ripple mode detection to work (along with the discovery of the ripple.config file). I believe we just about have that sorted out now.

I'll let @andreasohlund or @SimonCropp chime in on that one.

**Comment by danielmarbach** [comment]
If I remember correctly this has nothing todo with the msbuild task. Andreas and Simon did monkeypatch (sorry guys couldn't resist ;)) their fork of ripple. The task is seperate.

**Comment by andreasohlund** [comment]
Correct, the "auto downgrade stability to released only if not on develop branch" is a separate thing
